crates/burn-dataset/src/vision/image_folder.rs

```diff
@@ -82,7 +85,7 @@ pub struct SegmentationMask {
 /// Object detection bounding box annotation.
 #[derive(Deserialize, Serialize, Debug, Clone, PartialEq)]
 pub struct BoundingBox {
-    /// Coordinates.
+    /// Coordinates in [x,y,width,height] format.
```

Comment:
Just double checked the format, and for clarity I think I would be even more precise by stating `[x_min, y_min, width, height]` since there also exists `[cx, cy, width, height]` which could be confusing.

Reply:
Oops, I actually missed that, was not aware of the `_min` thing at all, will update the comment.

Reply:
Looks like the comment wasn't updated with the last changes 🤔